nodebuilder/state/state.go

```diff
@@ -150,11 +150,11 @@ type API struct {
 		QueryDelegation func(
 			ctx context.Context,
 			valAddr state.ValAddress,
-		) (*types.QueryDelegationResponse, error) `perm:"public"`
+		) (*types.QueryDelegationResponse, error) `perm:"read"`
 		QueryUnbonding func(
 			ctx context.Context,
 			valAddr state.ValAddress,
-		) (*types.QueryUnbondingDelegationResponse, error) `perm:"public"`
+		) (*types.QueryUnbondingDelegationResponse, error) `perm:"read"`
```

Comment:
Why not public?

Reply:
True. We should maybe think about making these methods take the AccountAddress as well